src/index.html

```diff
@@ -5,8 +5,443 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto&display=swap" rel="stylesheet">
   </head>
-  <body>
-    <h1>Catalog</h1>
+  <body class="catalog-page">
+    <header class="header catalog-page__header">
+      <a href="#home" class="header__logo">
+        <img src="./images/logo.png" alt="logoMoyo"
+        >
+      </a>
+
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a class="nav__link is-active" href="#apple">APPLE</a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link" href="#samsung">SAMSUNG</a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link" href="#phones">СМАРТФОНЫ И ТЕЛЕФОНЫ</a>
+          </li>
+          <li class="nav__item">
+            <a
+              class="nav__link"
+              href="#laptops"
+              data-qa="nav-hover"
+            >НОУТБУКИ И КОМПЬЮТЕРЫ</a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link" href="#gadgets">ГАДЖЕТЫ</a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link" href="#tablets">ПЛАНШЕТЫ</a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link" href="#photo">ФОТО</a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link" href="#video">ВИДЕО</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main class="catalog catalog-page__catalog">
+      <section class="card catalog__card" data-qa="card">
+        <img
+        src="./images/imac.jpeg"
+        alt="iMac image"
+        class="card__image"
+      >
+
+        <section class="card__product-name">
+          <h1 class="card__product-title">
+            Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)
+          </h1>
+        </section>
+
+        <h3 class="card__product-number">
```

Comment:
You can use <p> instead of <h1>-<h3>.
But if it's a title of the card then <h1>